.github/workflows/unit-tests.yml

```diff
@@ -42,11 +42,7 @@ jobs:
           RUSTDOCFLAGS: '-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Zpanic_abort_tests'
 
       - name: Install grcov
-        uses: actions-rs/install@v0.1
-        with:
-          crate: grcov
-          version: latest
-          use-tool-cache: true
+        run: cargo install --locked grcov
```

Comment:
I think it is the easiest way to fix it.

Reply:
How to fix it with the failed PR, let them merge with the main branch?